src/raven.js

```diff
@@ -310,9 +310,11 @@ Raven.prototype = {
      * @param {object} options A specific set of options for this error [optional]
      * @return {Raven}
      */
-    captureException: function(ex, options) {
+    captureException: function(ex, options, skipframes) {
+        skipframes = skipframes || 0;
+
         // If not an Error is passed through, recall as a message instead
-        if (!isError(ex)) return this.captureMessage(ex, options);
+        if (!isError(ex)) return this.captureMessage(ex, options, skipframes + 1);
```

Comment:
So, by incrementing and passing `skipframes` to `captureMessage`, we can generate a new stack by throwing an error from *inside* `captureMessage`, slice off `skipframes` frames, and then we can send that w/ the payload to Sentry. As in, we can now attach valid stack traces alongside `captureMessage` calls.

I was going to do that in a separate PR.